src/components/Contentful/MediaItemsCentered.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+	<section-with-background-classic
+		:background-content="sectionBackground"
+		:vertical-padding="verticalPadding"
+	>
+		<template #content>
+			<kv-page-container>
+				<kv-grid
+					:style="customGridStyles"
+					class="tw-grid-cols-12 lg:tw-grid-cols-10 lg:tw-gap-y-4
+						tw-justify-items-center tw-text-center tw-rounded tw-bg-white
+						tw-py-5 lg:tw-px-[177px]"
+				>
+					<span
+						v-for="(item, index) in contentfulMediaItems"
+						:key="index"
+						class="tw-flex tw-justify-items-center tw-items-center tw-mb-2"
+						:class="[itemClasses, {
+							'md:tw-col-span-12': index === 9
```

Comment:
Is this necessary?

Reply:
This is a temporary measure that relates to the number of sponsors we have for the current campaign. Def. part of the layout revisit that Ryan mentioned. We ultimately want this to support an arbitrary number of media items and handle the layout without specific intervention.